src/tests/unit/tests/common/components/__snapshots__/issue-filing-needs-settings-help-text.test.tsx.snap

```diff
@@ -0,0 +1,25 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`IssueFilingNeedsSettingsHelpTextTest render with isOpen = false 1`] = `null`;
+
+exports[`IssueFilingNeedsSettingsHelpTextTest render with isOpen = true 1`] = `
+<div
+  aria-live="polite"
+  className="create-bug-button-help"
+  role="alert"
+>
+  Go to Settings to configure issue filing.
+</div>
+`;
+
```

Comment:
Are the snapshots on lines 15-25 obsolete versions from before the Filinig typo was fixed?

Reply:
Right, I just updated.